amundsen_application/static/css/_icons.scss

```diff
@@ -55,6 +60,8 @@ span.icon {
 }
 
 img.icon {
+  /*DEPRECATED: follow behavior above to generate
```

Comment:
nit: In favor of filing issues on the open source issues vs. leaving comments that end up being vague for people without context. So I'd suggest we file an issue for this concept, and then if we do feel strongly about leaving a comment, we can leave the link to the issue.

Reply:
I created an issue to have these all be refactored, in the meanwhile I think the comment might be necessary so that of anyone else is adding an icon before the issue is fixed they don't make the same mistake.